src/ucp/core/ucp_mm.c

```diff
@@ -293,21 +293,38 @@ ucs_status_t ucp_mem_unmap(ucp_context_h context, ucp_mem_h memh)
     return status;
 }
 
-static unsigned ucp_madvise2uct(unsigned flags) 
+static ucs_status_t ucp_advice2uct(unsigned ucp_advice, unsigned *uct_advice) 
 {
-    return flags;
+    ucs_status_t status;
+
+    status = UCS_OK;
+
+    switch(ucp_advice) {
+    case UCP_MADV_NORMAL:
+        *uct_advice = UCT_MADV_NORMAL;
+        break;
```

Comment:
can just 'return OK' or 'return INVALID_PARAM' in every 'case', and get rid of 'status' variable